File: commons/dto/src/main/java/net/blueshell/dto/PictureDTO.java
package net.blueshell.dto;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = false)
public class PictureDTO extends BaseDTO {
    private Long id;
    private String name;
    private String url;
    private long uploaderId;
    private long eventId;
}

File: api/src/main/kotlin/net/blueshell/api/feature/file/dto/PictureDTO.kt
package net.blueshell.api.feature.file.dto

import io.swagger.v3.oas.annotations.media.Schema
import net.blueshell.api.shared.dto.AuditedAutoIdDTO

@Schema(name = "Picture")
data class PictureDTO(
    var name: String? = null,
    var url: String? = null,
    var uploaderId: Long = 0,
    var eventId: Long = 0
) : AuditedAutoIdDTO()
